build/images/scripts/start_ovs_windows_containerd.ps1

```diff
@@ -0,0 +1,146 @@
+$ErrorActionPreference = "Stop"
+$OVSVersion = "3.0.5-antrea.0"
+$OVSDownloadURL = "https://downloads.antrea.io/ovs/ovs-$OVSVersion-win64.zip"
+$OVSPublishedHash = 'fd27703ef7314b26b98cffb7aea27d569530ebd3ac3c98daa981ca2654373032'
+$WorkDir = [System.IO.Path]::GetDirectoryName($myInvocation.MyCommand.Definition)
+$OVSDownloadDir = $WorkDir
+$OVSInstallDir = "C:\openvswitch"
+$OVSZip = "$OVSDownloadDir\ovs-win64.zip"
+$ImportCertificate = $true
+$CheckFileHash = $true
+$PowerShellModuleBase = "C:\Windows\System32\WindowsPowerShell\v1.0\Modules"
+$InstallLog = "$OVSDownloadDir\install_ovs.log"
+
+function WaitExpandFiles($Src, $Dest) {
+    Write-Host "Extract $Src to $Dest"
+    Expand-Archive -Path $Src -DestinationPath $Dest | Out-Null
+}
+
+function CheckIfOVSInstalled() {
+    if (Test-Path -Path $OVSInstallDir) {
+        Write-Host "$OVSInstallDir already exists, OVS is already installed."
+        return $true
+    }
+    else {
+        Write-Host "$OVSInstallDir does not exist, OVS is not installed."
+        return $false
+    }
+}
+
+
+
+function DownloadOVS() {
+    If (!(Test-Path $OVSDownloadDir)) {
+        mkdir -p $OVSDownloadDir
+    }
+    Write-Host "Downloading OVS package from $OVSDownloadURL to $OVSZip"
+    curl.exe -sLo $OVSZip $OVSDownloadURL
+    If (!$?) {
+        Write-Host "Download OVS failed, URL: $OVSDownloadURL"
+        exit 1
+    }
+    if ($CheckFileHash) {
+        $FileHash = Get-FileHash $OVSZip
+        If ($OVSPublishedHash -ne "" -And $FileHash.Hash -ne $OVSPublishedHash) {
+            Write-Host "SHA256 mismatch for OVS download"
+            exit 1
+        }
+    }
+
+    Write-Host "Download OVS package success."
+}
+
+function InstallOVS() {
+    WaitExpandFiles $OVSZip $OVSDownloadDir
+    # Copy OVS package to target dir.
+    Write-Host "Copying OVS package from $OVSDownloadDir\openvswitch to $OVSInstallDir"
+    mv "$OVSDownloadDir\openvswitch" $OVSInstallDir
+    rm $OVSZip
+    # Create log and run dir.
+    $OVS_LOG_PATH = $OVSInstallDir + "\var\log\openvswitch"
+    If (!(Test-Path $OVS_LOG_PATH )) {
+        mkdir -p $OVS_LOG_PATH 
+    }
+    $OVSRunDir = $OVSInstallDir + "\var\run\openvswitch"
+    If (!(Test-Path $OVSRunDir)) {
+        mkdir -p $OVSRunDir 
+    }  
+}
+
+function InstallDriver(){
+   $OVSDriverDir = "$OVSInstallDir\driver"
+   # Install OVS driver certificate.
+   $DriverFile="$OVSDriverDir\OVSExt.sys"
+   if ($ImportCertificate) {
+       $CertificateFile = "$OVSDriverDir\package.cer"
+       if (!(Test-Path $CertificateFile)) {
+            $ExportType = [System.Security.Cryptography.X509Certificates.X509ContentType]::Cert;
```

Comment:
For signed drivers, we do not need to import certificates. For unsigned drivers, users should complete the certificate installation on the host in advance. Therefore, in any case, we do not need to include this operation in initContainer behavior.

Reply:
Please also update document for the unsigned driver use case. We don't support installing unsigned driver in container.

Reply:
Reminder